test/integrations/validation/_fixtures/Validator.fixtures.ts

```diff
@@ -10,6 +10,8 @@ const INVALID_DATE = 'Invalid date';
 const INVALID_ID = 'Invalid id';
 const INVALID_EMAIL = 'Invalid email';
 const INVALID_LIST = 'Invalid list';
+const INVALID_URL = 'Invalid url';
+const TOO_LONG_URL = 'https://' + 'example.com/images'.repeat(115) + '/peter.jpg';
```

Comment:
This url has been moved here - which is fine - but it still contains information from the masking URL and isn't properly constructed.

```suggestion
const TOO_LONG_URL = 'https://example.com/' + 'very_long_folder_name/'.repeat(115);
```